M_DD.H

```diff
@@ -1,14 +1,22 @@
 // m_dd.h
+#if defined(_WIN64) || !defined(_WIN32)
+#include <SDL_pixels.h>
+#endif
 
 class pic8;
 class palyaleiro;
 class recorder;
 
 class ddpal {
     friend void lassufizre(pic8* ppic, ddpal* ppal);
+#if defined(_WIN64) || !defined(_WIN32)
+    SDL_Color pal[256];
+#endif
 
   public:
+#if defined(_WIN32) && !defined(_WIN64)
     void* pddpal;
```

Comment:
This is pddpal and the other is pal? How does that work :bear: not sure I understand the intent of this commit clearly so I ask for clarification.

Reply:
Because pddpal is used by `F_WIN/W_DD.CPP` and `pal` by SDL.CPP.